src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfSource = sourceString
+    .split(';')
+    .filter((part) => part.trim() !== '');
+  const MappedArrayOfSource = arrayOfSource.map((part) => {
+    return part.split(':');
+  });
+  const result = {};
```

Comment:
The variable name 'result' is too generic. Use a more descriptive name, such as 'styleObject' to improve readability.